### ISSUE TITLE
About Us

### ISSUE THREAD

**Issue by singularityhacker** [ken333135/bankless-website#7]

**Comment by singularityhacker** [comment]
@roseja Will you be able to finish this page today? Could you push the newest and ask someone else to finish if not?

**Comment by roseja** [comment]
> @roseja Will you be able to finish this page today? Could you push the newest and ask someone else to finish if not?

No, there is a lot of extra features needed and I'm not that comfortable in the frontend.  I didn't see a comment back in the discord chat over the weekend. I can push what is there, I did have the PR up already like I said I would over the weekend. I wasn't sure if anyone else was going to merge that in. I'll look at that again now.

**Comment by roseja** [comment]
@singularityhacker Here is the PR that I had before but looks like there is a conflict now: https://github.com/ken333135/bankless-website/pull/50

**Comment by singularityhacker** [comment]
Change text to say something like "What is BanklessDAO"? [https://i.imgur.com/rPxrB8Y.png](https://i.imgur.com/rPxrB8Y.png)

**Comment by singularityhacker** [comment]
Links under the main image: Remove "coordination" and "media kit" and hyperlink the rest of them (5 links).

**Comment by singularityhacker** [comment]
Still a responsive issue with this page at small sizes.

**Comment by singularityhacker** [comment]
change the video at the bottom to something cool like this: [https://opensea.io/assets/0x1392e2a50a64c03e418c063358360f7639756c2e/1](https://opensea.io/assets/0x1392e2a50a64c03e418c063358360f7639756c2e/1)

**Comment by ken333135** [comment]
@sunshine1004-dev hey Olek, can you help with the mobile responsiveness for the bottom half of this page? the font sizes arent scaling properly. thank you!
